src/styles/scss/_gjs_layers.scss

```diff
@@ -100,13 +113,10 @@ $layerNameSpacing: 5px !default;
     }
 
     &-name {
-      padding: $layerNameSpacing 0;
       display: inline-block;
       box-sizing: content-box;
-      overflow: hidden;
-      white-space: nowrap;
```

Comment:
Any reason behind removing these? Those properties are used to apply ellipsis to long layer names.

Reply:
Thank you very much on your feedback @artf :)

It's dark-on-dark and I'm blind as a bat, so didn't see the border underneath. Will fix this up and the CSS code for the ellipsis. I'll resubmit once I've got it updated.

Reply:
Hi @artf 

I've made the requested fixes. Please let me know if you require anything further.

Thank you very much Artur